lib/src/components/card/Card.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { bool, func, string } from 'prop-types';
+import cx from 'classnames';
+import { CardContent } from './CardContent';
+import { CardControls } from './CardControls';
+
+function Card({
+  onClick,
+  className,
+  children,
+  innerClassNames,
+  selected,
+  highlighted,
+  added,
+  removed,
+  disabled
+}) {
+  const classNames = cx('card outline-none', className, {
+    'card-interactive': onClick,
+    'card-selected': selected,
+    'card-highlighted': highlighted,
+    'card-added': added,
+    'card-removed': removed,
+    'card-disabled': disabled
+  });
+
+  const innerClasses = cx('card-inner', innerClassNames);
+
+  const handleEditKeyPress = e => {
+    if (e.key === 'Enter') {
+      onClick(e);
+    }
+  };
+
+  return (
+    <div // eslint-disable-line jsx-a11y/no-static-element-interactions
+      role={onClick ? 'button' : 'presentation'}
+      className={classNames}
+      onClick={onClick}
+      onKeyUp={handleEditKeyPress}
```

Comment:
🚩 A really soft red flag here just because this confused me!

What's this for? Just looking at card I don't know what "edit" is and why we have a keypress that fires onclick. Is it also possible that I wouldn't want enter to fire onClick?

Reply:
ESLint requires it. Any "none" interactive element which takes an interactive role needs to handle keyUp events (which are only fired when you are focused on the element in the quest and press any key.

It's also worth noting that the original card component did this, I have just moved the card component into the new `src/components` DIR and refactored it to be functional 👍 

----

I will rename it though so it's just `handleKeyPress` again this was just copied so it kept the name 🤦